features/parameter-types/rooms.js

```diff
@@ -33,7 +33,7 @@ class Room {
 // See: https://github.com/zinc-collective/convene/issues/41
 defineParameterType({
   name: "accessLevel",
-  regexp: /(an |a |the )?(Unlocked|Internal|Locked)/,
+  regexp: /(an |a |the |is )?(Unlocked|Internal|Locked)/,
```

Comment:
Think of a better way to solve this.

Reply:
I think we may want a set of article adjectives that are shared across the parameter types; perhaps there's a way to interpolate a constant into a regex? (Dunno if this syntax is correct; but this is the rough shape)

```javascript
const FLEXIBLE_ARTICLE_ADJECTIVES="( |an |the |is |a )"

// ...
defineParameterType({
  //...
  regexp: /${FLEXIBLE_ARTICLE_ADJECTIVES}(Unlocked|Internal|Locked)/
})
```